src/test/java/bio/terra/service/filedata/google/firestore/FileOperationTest.java

```diff
@@ -23,12 +23,20 @@
 import bio.terra.model.FileModel;
```

Comment:
@fboulnois  @ddietterich Definitely open to suggestions if y'all think I need more test coverage.

Reply:
I think this is great!